datanode/api/trading_data_v2.go

```diff
@@ -243,6 +245,51 @@ func (t *tradingDataServiceV2) ListLedgerEntries(ctx context.Context, req *v2.Li
 	}, nil
 }
 
+func (t *tradingDataServiceV2) ExportLedgerEntries(ctx context.Context, req *v2.ExportLedgerEntriesRequest) (*v2.ExportLedgerEntriesResponse, error) {
+	defer metrics.StartAPIRequestAndTimeGRPC("ExportLedgerEntriesV2")()
+	if t.accountService == nil {
+		return nil, apiError(codes.Internal, ErrAccountServiceSQLStoreNotAvailable, nil)
+	}
+
+	dateRange := entities.DateRangeFromProto(req.DateRange)
+	pagination, err := entities.CursorPaginationFromProto(req.Pagination)
+	if err != nil {
+		return nil, apiError(codes.InvalidArgument, fmt.Errorf("invalid cursor: %w", err))
+	}
+
+	raw, pageInfo, err := t.ledgerService.Export(ctx, req.PartyId, req.AssetId, dateRange, pagination)
+	if err != nil {
+		apiError(codes.Aborted, err)
+	}
+
+	md, ok := metadata.FromIncomingContext(ctx)
+	if !ok {
+		return nil, apiError(codes.DataLoss, fmt.Errorf("failed to get context metadata"))
+	}
```

Comment:
I'm not sure what this part is for, xrid doesn't seem to be used?

Reply:
This has been removed, it uses the gRPCGateway header now
Thanks!